modules/understanding-upgrade-channels.adoc

```diff
@@ -134,5 +134,21 @@ ifndef::openshift-origin[]
 [discrete]
 == Switching between channels
 
-Your cluster is still supported if you change from the `stable-{product-version}` channel to the `fast-{product-version}` channel. Although you can switch to the `candidate-{product-version}` channel at any time, some releases in that channel might be unsupported release candidates. You can switch from the `candidate-{product-version}` channel to the `fast-{product-version}` channel if your current release is a general availability release. You can always switch from the `fast-{product-version}` channel to the `stable-{product-version}` channel, although if the current release was recently promoted to `fast-{product-version}` there can be a delay of up to a day for the release to be promoted to `stable-{product-version}`. If you change to a channel that does not include your current release, an alert displays and no updates can be recommended, but you can safely change back to your original channel at any point.
+A channel can be switched from the web console or through the `adm upgrade channel` command:
+
+----
+$ oc adm upgrade channel clusterversion version --type json -p '[{"op": "add", "path": "/spec/channel", "value": "<channel>”}]'
+----
+
+If you change to a channel that does not include your current release, an alert will display and no updates can be recommended, but you can safely change back to your original channel at any point.
```

Comment:
I'd revise this sentence to be a bit more concise since there are three independent clauses and one dependent clause. I think the middle two clauses are also passive voice.

Maybe:
> The web console displays an alert if you switch to a channel that does not include the current release. The web console does not recommend any updates while on a  channel without the current release. You can return to the original channel at any point, however.